### ISSUE TITLE
Trouble Authenticating Wifi

### ISSUE THREAD

**Issue by audreyobscura** [bdring/DrawBot_Badge#1]
Hey there! Thanks again for putting together this awesome little CNC badge. I've been looking for ways to get started prototyping small-scale CNC machines and was elated to hear of your workshop. 

I followed along with the updated troubleshooting connectivity issues in the Wiki and am still a bit unsure why I'm not able to authenticate the on to the network. 

I am connected to a 12v 5a power supply and have flashed the board with the most recent build of your code, and I'm still getting the same Invalid Password error. Here is a view of my terminal and the error messages I'm getting.  

I also had this trouble during our workshop, I was able to authenticate one time with my phone, and have been locked out since.

![screen shot 2018-11-08 at 3 40 54 pm](https://user-images.githubusercontent.com/5462048/48234552-e4d6f300-e36e-11e8-9217-9bba0d728ba8.png)
![screen shot 2018-11-08 at 3 17 08 pm](https://user-images.githubusercontent.com/5462048/48234553-e4d6f300-e36e-11e8-8f7f-81181cdb1b32.png)
![screen shot 2018-11-08 at 3 16 55 pm](https://user-images.githubusercontent.com/5462048/48234554-e4d6f300-e36e-11e8-89b2-88aaa2870009.png)

Let me know if you need any additional documentation, and I'll keep crawling the ESP32 forums to see if I can find out why this may be happening.

Thanks!

**Comment by bdring** [comment]
It could be something wrong with the ESP32. I had to go into each badge via wifi to setup the web pages and change the SSIDs and password.

I am about to order some very cheap versions of the board, that do badge functions only. If you are still having trouble when those arrive, I'll send you a tested one.

**Comment by audreyobscura** [comment]
Yeah, I've tried everything. It's a weird error to have but I've talked to some folks at work (Instructables) and they suspect its an issue with the ESP32. Sometimes it happens! Thanks for encouraging me to keep testing, I definitely am learning a lot!
